scripts/templates/component-story.js

```diff
@@ -1,7 +1,6 @@
 module.exports = componentName => [
   {
     content: `// Generated with scripts/create-componnent.js
-import React from 'react';
 import {
   AutoExample,
```

Comment:
Shouldnt this story be updated in new template of docs from storybook?
In this example, the storybook-ui still in use.